astropy/cosmology/_src/flrw/base.py

```diff
@@ -80,6 +80,45 @@ def quad(*args, **kwargs):
 _FLRWT = TypeVar("_FLRWT", bound="FLRW")
 _FlatFLRWMixinT = TypeVar("_FlatFLRWMixinT", bound="FlatFLRWMixin")
 
+
+##############################################################################
+
+
+class NeutrinoInfo(NamedTuple):
+    """A container for neutrino information.
+
+    This is Private API.
+
+    """
+
+    nneutrinos: int
```

Comment:
GIven the other names, I'd have expected `n_nu` or perhaps `n_neutrino`.  Are you sure you want this name?

Reply:
Agreed! I'll change it.

Reply:
Done